src/Routes/Devices/DeviceTable.js

```diff
@@ -10,9 +10,14 @@ import CustomEmptyState from '../../components/Empty';
 import { useHistory } from 'react-router-dom';
 import { emptyStateNoFliters } from '../../utils';
 import DeviceStatus from '../../components/Status';
+import RetryUpdatePopover from './RetryUpdatePopover';
 
-const getDeviceStatus = (deviceStatus, isUpdateAvailable) =>
-  deviceStatus === 'UPDATING'
+const getDeviceStatus = (deviceStatus, isUpdateAvailable, DispatcherStatus) =>
```

Comment:
For `DispatcherStatus`, `DeviceStatus` and any other variables that are created on our end lets use the camelCase convention. :smile: